cpp/src/centrality/betweenness_centrality.cu

```diff
@@ -300,6 +312,11 @@ void BC<VT, ET, WT, result_t>::add_reached_endpoints_to_source_betweenness(VT so
 template <typename VT, typename ET, typename WT, typename result_t>
 void BC<VT, ET, WT, result_t>::add_vertices_dependencies_to_betweenness()
 {
+  thrust::host_vector<result_t> h_betweenness(number_of_vertices_, 0);
+  result_t *ptr = h_betweenness.data();
+  CUDA_TRY(
+    cudaMemcpy(ptr, betweenness_, number_of_vertices_ * sizeof(result_t), cudaMemcpyDeviceToHost));
```

Comment:
Better use cudaMemcpyAsync with ```stream_``` to be stream-ready

Reply:
Look for https://github.com/rapidsai/raft/blob/branch-0.15/cpp/include/raft/cudart_utils.h#L131

Reply:
Updated in eab569b to use `cudaMemcpyAsync`